### ISSUE TITLE
The loss of rpn net

### ISSUE THREAD

**Issue by tendence** [linmx0130/ya_mxdet#4]
I remember in paper the loss should be loss = loss_cls + lambda*loss_reg instead of loss = loss_cls + loss_reg

why u changed to this?

**Comment by linmx0130** [comment]
I just implemented in a simplest way. 
In fact in the original Faster RCNN Python repo, lambda=1 for RPN, but the loss_reg for RCNN is a little tricky. That configure will improve the prediction result indeed.
